docs/Webhooks/Creating_subscription.rst

```diff
@@ -0,0 +1,55 @@
+.. _creating_subscription:
+
+Creating webhook subscription (Beta version)
+============================================
+
+To create a webhook subscription, send a ``POST`` request to ``/webhooks``.
+The body needs to contain the following JSON data. For example:
+
+  .. code-block:: JSON
+   :linenos:
+    {
+        "description": "Subscription for events",
+        "url": "https://ennpjtc2q1cm.x.pipedream.net",
+        "events": [
+            "OrderCreated"
+        ]
+    }
+
+The response, if successfully will be a JSON object with the following data:
+
+  .. code-block:: JSON
+   :linenos:
+
+    {
+        "success": true
```

Comment:
I know this is up to personal taste, but a 200 status code is already "ok", so in my personal opinion the success = true is a bit redundand.

Reply:
yeah, I think usually response is expected when an endpoint is hit and the status code is checked only later.

Reply:
Hi, it would be nice to have 201 Created instead of 200 OK, as webhook subscription was created.

Maybe in this case it would be nice to add some configuration body instead of `success: true`. You can use object from GET / endpoint if you have some

Reply:
Thank you, Oleksii for reviewing and giving us suggestions. will take them into consideration :)